scripts/submitHistoFiller_lxplus.py

```diff
@@ -0,0 +1,147 @@
+import os
+import sys
+import argparse
+
+
+write = lambda stream, text: stream.write(text + '\n')
+
+
+def make_parser():
+    parser = argparse.ArgumentParser(description='Command line parser of plotting options')
+    parser.add_argument('--outdir', type=str, dest="outdir", required=True,
+                        help='/afs/directory/to/submit/from. Must be afs\
+    because condor submission is only allowed from /afs on lxplus.')
+    parser.add_argument('--cfg', dest='cfg', required=True,
+                        help='name of the main cfg to run', default=None)
+    parser.add_argument('--tag', dest='tag', required=True, default=None,
```

Comment:
again IIUC: `tag` is what I would have naively called `outdir` (ie the directory where the histograms should be saved as defined in the `mainCfg`), right?

Reply:
Yeah exactly. Implemented this change in the latest commit.